inspirehep/ext/redis/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
Unless something has changed, it might still be fine actually. We have it everywhere nowadays.